snuba/migrations/parse_schema.py

```diff
@@ -72,18 +72,21 @@ def merge_modifiers(
         return col_type.set_modifiers(existing_modifiers.merge(modifiers))
 
 
+COLUMN_MAPPERS: Mapping[str, type[ColumnType[MigrationModifiers]]] = {
```

Comment:
Since the dictionary was being built every time, it wasn't possible to give it a type without creating a variable for it that could then have a type. If I was going to do that, I might as well move it out of the function so the dictionary isn't being rebuilt over and over.